--plot_full_ref collides with plot_pose_correspondences

**Description:**
Pose correspondences should be plotted with the synced trajectory, not the full reference trajectory.

**Command:**
```
evo_config set plot_pose_correspondences true
evo_{ape, rpe} ... --plot_full_ref
```

**Console output:**
```
# blabla

[ERROR] trajectories must have same length to draw pose correspondences - try to synchronize them first
```

* evo version number shown by `evo pkg --version`: 1.9

--- a/evo/common_ape_rpe.py
+++ b/evo/common_ape_rpe.py
@@ -91,7 +91,7 @@ def get_delta_unit(args):
     return delta_unit
 
 
-def plot(args, result, traj_ref, traj_est):
+def plot(args, result, traj_ref, traj_est, traj_ref_full=None):
     from evo.tools import plot
     from evo.tools.settings import SETTINGS
 
@@ -123,7 +123,8 @@ def plot(args, result, traj_ref, traj_est):
     if args.ros_map_yaml:
         plot.ros_map(ax, args.ros_map_yaml, plot_mode)
 
-    plot.traj(ax, plot_mode, traj_ref, style=SETTINGS.plot_reference_linestyle,
+    plot.traj(ax, plot_mode, traj_ref_full if traj_ref_full else traj_ref,
+              style=SETTINGS.plot_reference_linestyle,
               color=SETTINGS.plot_reference_color, label='reference',
               alpha=SETTINGS.plot_reference_alpha)
     plot.draw_coordinate_axes(ax, traj_ref, plot_mode,

--- a/evo/main_ape.py
+++ b/evo/main_ape.py
@@ -243,9 +243,8 @@ def run(args):
     )
 
     if args.plot or args.save_plot or args.serialize_plot:
-        common.plot(args, result,
-                    traj_ref_full if args.plot_full_ref else traj_ref,
-                    result.trajectories[est_name])
+        common.plot(args, result, traj_ref, result.trajectories[est_name],
+                    traj_ref_full=traj_ref_full)
 
     if args.save_results:
         logger.debug(SEP)

--- a/evo/main_rpe.py
+++ b/evo/main_rpe.py
@@ -277,9 +277,8 @@ def run(args):
     )
 
     if args.plot or args.save_plot or args.serialize_plot:
-        common.plot(args, result,
-                    traj_ref_full if args.plot_full_ref else traj_ref,
-                    result.trajectories[est_name])
+        common.plot(args, result, traj_ref, result.trajectories[est_name],
+                    traj_ref_full=traj_ref_full)
 
     if args.save_results:
         logger.debug(SEP)

